--- atom-application.coffee.orig	2015-06-11 15:20:35.000000000 +0100
+++ atom-application.coffee	2015-06-11 15:36:35.000000000 +0100
@@ -17,9 +17,9 @@

 DefaultSocketPath =
   if process.platform is 'win32'
-    '\\\\.\\pipe\\atom-sock'
+    '\\\\.\\pipe\\prob2-sock'
   else
-    path.join(os.tmpdir(), "atom-#{process.env.USER}.sock")
+    path.join(os.tmpdir(), "prob2-#{process.env.USER}.sock")

 # The application's singleton class.
 #
@@ -71,9 +71,10 @@
     @pidsToOpenWindows = {}
     @pathsToOpen ?= []
     @windows = []
+    @probVersion = '#{probVersion}'

-    @autoUpdateManager = new AutoUpdateManager(@version, options.test)
-    @applicationMenu = new ApplicationMenu(@version, @autoUpdateManager)
+    @autoUpdateManager = new AutoUpdateManager(@probVersion, options.test)
+    @applicationMenu = new ApplicationMenu(@probVersion, @autoUpdateManager)
     @atomProtocolHandler = new AtomProtocolHandler(@resourcePath, @safeMode)

     @listenForArgumentsFromNewProcess()
@@ -178,13 +179,13 @@
       atomWindow ?= @focusedWindow()
       atomWindow?.browserWindow.inspectElement(x, y)

-    @on 'application:open-documentation', -> require('shell').openExternal('https://atom.io/docs/latest/?app')
-    @on 'application:open-discussions', -> require('shell').openExternal('https://discuss.atom.io')
+    @on 'application:open-documentation', -> require('shell').openExternal('http://stups.hhu.de/ProB/w/Main_Page')
+    @on 'application:open-discussions', -> require('shell').openExternal('http://stups.hhu.de/ProB/w/Main_Page')
     @on 'application:open-roadmap', -> require('shell').openExternal('https://atom.io/roadmap?app')
     @on 'application:open-faq', -> require('shell').openExternal('https://atom.io/faq')
-    @on 'application:open-terms-of-use', -> require('shell').openExternal('https://atom.io/terms')
-    @on 'application:report-issue', -> require('shell').openExternal('https://github.com/atom/atom/blob/master/CONTRIBUTING.md#submitting-issues')
-    @on 'application:search-issues', -> require('shell').openExternal('https://github.com/issues?q=+is%3Aissue+user%3Aatom')
+    @on 'application:open-terms-of-use', -> require('shell').openExternal('http://stups.hhu.de/ProB/w/Main_Page')
+    @on 'application:report-issue', -> require('shell').openExternal('https://probjira.atlassian.net')
+    @on 'application:search-issues', -> require('shell').openExternal('https://probjira.atlassian.net/browse/PROBPLUGIN-115?filter=11300')

     @on 'application:install-update', => @autoUpdateManager.install()
     @on 'application:check-for-update', => @autoUpdateManager.check()
