diff --git a/build/tasks/prebuild-less-task.coffee b/build/tasks/prebuild-less-task.coffee
index 5e3bd27..046457e 100644
--- a/build/tasks/prebuild-less-task.coffee
+++ b/build/tasks/prebuild-less-task.coffee
@@ -32,37 +32,7 @@ module.exports = (grunt) ->
     compileBootstrap()
 
     prebuiltConfigurations = [
-      ['atom-dark-ui', 'atom-dark-syntax']
-      ['atom-dark-ui', 'atom-light-syntax']
-      ['atom-dark-ui', 'one-dark-syntax']
-      ['atom-dark-ui', 'one-light-syntax']
-      ['atom-dark-ui', 'solarized-dark-syntax']
       ['atom-dark-ui', 'base16-tomorrow-dark-theme']
-      ['atom-dark-ui', 'base16-tomorrow-light-theme']
-
-      ['atom-light-ui', 'atom-light-syntax']
-      ['atom-light-ui', 'atom-dark-syntax']
-      ['atom-light-ui', 'one-dark-syntax']
-      ['atom-light-ui', 'one-light-syntax']
-      ['atom-light-ui', 'solarized-dark-syntax']
-      ['atom-light-ui', 'base16-tomorrow-dark-theme']
-      ['atom-light-ui', 'base16-tomorrow-light-theme']
-
-      ['one-dark-ui', 'one-dark-syntax']
-      ['one-dark-ui', 'one-light-syntax']
-      ['one-dark-ui', 'atom-dark-syntax']
-      ['one-dark-ui', 'atom-light-syntax']
-      ['one-dark-ui', 'solarized-dark-syntax']
-      ['one-dark-ui', 'base16-tomorrow-dark-theme']
-      ['one-dark-ui', 'base16-tomorrow-light-theme']
-
-      ['one-light-ui', 'one-light-syntax']
-      ['one-light-ui', 'one-dark-syntax']
-      ['one-light-ui', 'atom-light-syntax']
-      ['one-light-ui', 'atom-dark-syntax']
-      ['one-light-ui', 'solarized-dark-syntax']
-      ['one-light-ui', 'base16-tomorrow-dark-theme']
-      ['one-light-ui', 'base16-tomorrow-light-theme']
     ]
 
     directory = path.join(grunt.config.get('atom.appDir'), 'less-compile-cache')
